clusters/app.ci/prow/03_deployment/sinker.yaml

```diff
@@ -39,11 +39,8 @@ items:
           volumeMounts:
           - name: service-account-token
             mountPath: /var/sa-token
-          - name: config
-            mountPath: /etc/config
-            readOnly: true
-          - name: job-config
-            mountPath: /etc/job-config
+          - name: release
+            mountPath: /etc/release
```

Comment:
https://github.com/openshift/release/pull/26348#discussion_r808772869 applies, but whatever method we end up using:

1. We can only mount the parts that we need (`ci-operator/jobs` and `core-services/prow/02_config/`)  using `subPath` using two mounts of the same volume, right?
2. One thing we need to check really well is how well does the watcher (that reloads configuration in Prow components when they change) play with this implementation. It was built in for a CM mount and assumes some CM or even `config-updater` internals, such as a `VERSION` file.